yt_dlp/extractor/glomex.py

```diff
@@ -0,0 +1,282 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import (
+    compat_urllib_parse_urlparse,
+    compat_urllib_parse_urlencode,
+)
```

Comment:
compat is not needed in dlp since we don't support py2

Reply:
Not sure if there's a policy/bcp for dlp, as these are still defined and can be imported `yt_dlp.compat`, but I went ahead and imported them directly from `urllib.parse`.

Reply:
Either is fine actually